### PR TITLE
Document known performance limitation for scoped admins using list users endpoints.

### PR DESCRIPTION
**Pull request by chenningzhang-okta**
## Description:
- **What's changed?** Document known performance limitation for scoped admins using list users endpoints.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-482887](https://oktainc.atlassian.net/browse/OKTA-482887)

@nghadge-okta @donwoods-okta @okta/ud-core

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -1141,6 +1141,11 @@ The first three parameters in the table below correspond to different types of l
 
 Array of [User](#user-object)
 
+##### Known Limitation
+
+Due to an infrastructure limitation, [group administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-group-admin.htm), [help desk administrators](https://help.okta.com/en-us/Content/Topics/Security/administrators-help-desk-admin.htm),
```
Comment (joshreesjones-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1146]: Can you describe the limitation (in the pull request, not the documentation)? Wondering if "infrastructure" is the right word choice.

**Review by donwoods-okta — APPROVED** [review]
LGTM

**Review by joshreesjones-okta — APPROVED** [review]

**Review by nghadge-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 1 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1146]: Need to use aliases for this links. I have updated.

```suggestion
Due to an infrastructure limitation, [group administrators](https://help.okta.com/okta_help.htm?id=ext_The_User_Admin_Role), [help desk administrators](https://help.okta.com/okta_help.htm?id=ext_The_Help_Desk_Admin_Role),
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 2]
```diff
@@ -1141,6 +1141,11 @@ The first three parameters in the table below correspond to different types of l
...
+and [custom administrators](https://help.okta.com/en-us/Content/Topics/Security/custom-admin-role/custom-admin-roles.htm) who are only scoped to view and manage users of their assigned groups may experience timeout for the list users endpoints.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:1147]:
```suggestion
and [custom administrators](https://help.okta.com/okta_help.htm?id=csh-cstm-admin-roles) who are only scoped to view and manage users of their assigned groups may experience timeout for the list users endpoints.
```

**Pull request merged**